views/projects/show.erb

```diff
@@ -3,43 +3,90 @@
     <a href="/projects/<%= @project.id %>/edit">edit</a>
   </h2>
   <p>
-    Client Id: <%= @project.client_id %> <br>
-    Resources: <%= @project.resources %> <br>
+    <% for @client in @clients %>
+      <% if @client.active %>
+      Client:
+        <a href="/clients/<%= @client.id %>" >
+          <%= @client.name %>
+        </a>
+      <% end %>
+    <% end %>
+
+    <br>
     Description: <%= @project.description %> <br>
-    Active: <%= @project.active %><br>
+    Active State: <%= @project.active %><br>
```

Comment:
It's a minor thing, but you could do something like
```
<%= @project.active ? "Active" : "Inactive" %>
```
To either print Active or Inactive depending on the field value. You could even have it print a span around the text with a green or red class to make it more obvious.